config/schema.yaml

```diff
@@ -0,0 +1,340 @@
+# Stage variables
```

Comment:
option ERROR_REPORT_DIR_NESTING_LEVEL is missed in this file
```BuildInterface::VAR_ERROR_REPORT_DIR_NESTING_LEVEL```

Reply:
There are bunch of them missed, will be added in next task

Reply:
Plus some other options are missed:
ELASTICSUITE_CONFIGURATION
CACHE_CONFIGURATION
DATABASE_CONFIGURATION
...

Reply:
Will be moved in next task